modules/oe_theme_helper/tests/src/Functional/RetinaScaleEffectTest.php

```diff
@@ -4,7 +4,8 @@
 
 namespace Drupal\Tests\oe_theme_helper\Functional;
 
-use Drupal\FunctionalTests\Image\ToolkitTestBase;
+use Drupal\Tests\token\Kernel\KernelTestBase;
```

Comment:
This doesn't look as the correct class to extend.

Reply:
Yes I know, I've already fixed it.